src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -804,11 +806,19 @@ public function Breadcrumbs($unlinked = false)
             $items = new ArrayList();
         }
 
+        $params = $this->getRequest()->getVars();
+        if (isset($params['url'])) {
+            unset($params['url']);
+        }
+
         if ($this->record && $this->record->ID) {
             $title = ($this->record->Title) ? $this->record->Title : "#{$this->record->ID}";
             $items->push(new ArrayData([
                 'Title' => $title,
-                'Link' => $this->Link()
+                'Link' => Controller::join_links(
+                    $this->Link(),
+                    '?' . http_build_query($params ?? [])
+                ),
```

Comment:
What are we doing this for?

Reply:
To keep request in Breadcrumbs links. Initially this logic lost request parameters.